frontend/views/pages/DesignSystem.vue

```diff
@@ -1254,6 +1279,11 @@ export default {
         el.link.classList.remove('last')
       })
       this.articles[last].link.classList.add('last')
+    },
+    async saveIn2500ms () {
```

Comment:
One last thing (might as well fix this since the other `L` -related string stuff needs to be fixed too): this is a minor detail, and I don't think it ultimately affects the behavior of the code, but when you are returning a promise like, and not using `await` inside the function, the `async` modifier should not be added to the function.

Having the `async` modifier there, in this case, will (I think), cause it to return a promise, that returns this promise, requiring two promise resolutions, which is just kinda unnecessary. So the `async` modifier isn't needed, since we're already returning a promise (and not calling `await` anywhere else).